docker/Dockerfile.windows

```diff
@@ -9,7 +8,7 @@ LABEL org.opencontainers.image.licenses="Apache-2.0"
 LABEL org.opencontainers.image.title="Chocolatey"
 LABEL org.opencontainers.image.description="Chocolatey Client running on .NET"
 LABEL org.opencontainers.image.authors="https://chocolatey.org/support"
-LABEL org.opencontainers.image.base.name="mcr.microsoft.com/windows/servercore:${tagversion}"
+LABEL org.opencontainers.image.base.name="mcr.microsoft.com/dotnet/framework/sdk:4.8-windowsservercore-ltsc2016"
```

Comment:
Is there a reason for this switch to the dotnet framework image? And is there a reason the sdk image is being used instead of the runtime image?

Reply:
The switch to the dotnet image is used is due to the dotnet 4.8 install during the installation step failing during the build, when using the base servercore image. It seemed like switching to an image that contained the required stuff was a reasonable move - however, we can absolutely trim down to runtime!